backend/coins/btc/headers/headers.go

```diff
@@ -180,7 +180,10 @@ func (headers *Headers) Initialize() {
 			headers.update(header.BlockHeight)
 		},
 	)
-	headers.kickChan <- struct{}{}
+	go func() {
```

Comment:
Why do we need the 30s sleep?

Reply:
we don't, it's a debug leftover :cold_sweat: thanks, undone!